fanficfare/flaresolverr_proxy.py

```diff
@@ -151,12 +151,13 @@ def request(self, method, url, headers=None, parameters=None):
                                # 428 ('Precondition Required') gets the
                                # error_msg through to the user.
             data = resp.json['message']
-        if status_code != 200 and status_code != 400: # 400 for scribblehub ajax query 
-                raise exceptions.HTTPErrorFFF(
-                    ensure_text(url),
-                    status_code,
-                    ensure_text(data)
-                    )
+        if status_code != 200:
+            raise exceptions.HTTPErrorFFF(
+                ensure_text(url),
+                status_code,
+                ensure_text(data),
+                ensure_text(data)
```

Comment:
I believe the data in 159 should be passed without `ensure_text()`?  Since it gets decoded in the adapter anyway?

Reply:
You're totally right, looks like this saved the bacon there - I'll remove the ensure_text() anyway and commit :) https://github.com/JimmXinu/FanFicFare/blob/2e192380f009809b8e350bbb78b84aa242a6ed45/fanficfare/requestable.py#L39